lib/controller/accountscontroller.php

```diff
@@ -231,7 +231,12 @@ public function send($accountId) {
 			$message = $mailbox->getMessage($messageId);
 
 			if (is_null($subject)) {
-				$headers['Subject'] = "RE: " . $message->getSubject();
+				// prevent 'Re: Re:' stacking
+				if(substr($message->getSubject(), 0, 4) === 'Re: ') {
```

Comment:
You might want to compare case insensitive here to catch usages of RE as well - http://php.net/manual/en/function.strcasecmp.php